src/main/java/seedu/exchangecoursemapper/command/ObtainContactsCommand.java

```diff
@@ -42,13 +48,27 @@ public void execute(String userInput) {
         logger.log(Level.INFO, Logs.COMPLETE_EXECUTION);
     }
 
+    /**
+     * Filters the school name from the user input.
```

Comment:
I'm not sure if I remembered correctly but I think it was recommended to start the JavaDocs with the word `Returns` if it is not a void method!

Example:
```
Returns the school name from the user input
```